spec.html

```diff
@@ -19527,9 +19527,6 @@ <h1>Static Semantics: Early Errors</h1>
         <li>
           It is a Syntax Error if |ArrowParameters| Contains |AwaitExpression| is *true*.
         </li>
-        <li>
-          It is a Syntax Error if ContainsUseStrict of |ConciseBody| is *true* and IsSimpleParameterList of |ArrowParameters| is *false*.
```

Comment:
How is this dead?  [`ContainsUseStrict` of `FunctionBody`](https://tc39.es/ecma262/#sec-function-definitions-static-semantics-containsusestrict) can return `true`, and [`ConciseBody` can produce `{ FunctionBody }`](https://tc39.es/ecma262/#prod-ConciseBody). By the [chain rule](https://tc39.es/ecma262/#sec-algorithm-conventions-syntax-directed-operations), then, `ContainsUseStrict of |ConciseBody|` can return `true`.

Same for the other early error, except with one more intermediate step in the chain rule (`AsyncFunctionBody` -> `FunctionBody`).

Reply:
Ah, I saw `ConciseBody : AssignmentExpression` and thought that was the only alternative, but forgot about the `{ FunctionBody }` case. Yeah, the chain rule should make the early errors not dead. I guess I'll just drop the second commit then and have this PR add the `ContainsUseStrict` definition for `AsyncConciseBody : AssignmentExpression`.